### ISSUE TITLE
[indent] wrong indentation on enums with allman style

### ISSUE THREAD

**Issue by spicard12** [typescript-eslint/typescript-eslint#3225]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/brace-style": ["error", "allman"],
    "@typescript-eslint/indent" : ["error", 4]
  }
}
```

```TS
// your repro code case
export enum Language
{
    Typescript = 'typescript',
    Cpp = 'cpp',
}
```

**Expected Result**

When running ESLint, it should not return any errors for concerning this particular piece of code.

Also, when running ESLint with the `--fix` flag, the code shouldn't be modified at all.

**Actual Result**

ESlint returned an error concerning this particular piece of code :
2:1  error  Expected indentation of 4 spaces but found 0  @typescript-eslint/indent

When running ESLint with the `--fix` flag, it will output this as the fixed version :

```TS
export enum Language
    {
    Typescript = 'typescript',
    Cpp = 'cpp',
}
```

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.19.0` |
| `@typescript-eslint/parser`        | `4.19.0` |
| `TypeScript`                       | `4.1.3` |
| `ESLint`                           | `7.19.0` |
| `node`                             | `14.15.4` |

**Comment by bradzacher** [comment]
Please use the search next time https://github.com/typescript-eslint/typescript-eslint/issues/1824

**Comment by spicard12** [comment]
Sorry about the duplicate. I had searched before creating the issue, but obviously, I didn't search for the right thing :(
